lib/mayaUsd/resources/ae/usdschemabase/material_custom_control.py

```diff
@@ -76,7 +76,7 @@ def _hasLookdevX():
         '''
```

Comment:
I didn't realize that we have soft dependency on LookdevX. Is this necessary or could we expose a callback for LookdevX to add their own ?